the first task.cpp

```diff
@@ -1,7 +1,9 @@
-#include <iostream>
+#include <iostream> // 2/2
 
-int Main()
+int main() // 0/1
 {
-	std::cout << "Hello, World!\n";
-		return 0;
-}
\ No newline at end of file
+	std::cout << "Hello, World!\n"; // 4/4
+	return 0; // 1/1
+}
+
+// great work 😎 9/10 ⭐

```

Comment:
- [x] most of code errors and bugs are fixed
- [ ] file is in the right path
- [x] +1 marks for your first commit